pkg/cluster/deploystorage.go

```diff
@@ -109,12 +109,21 @@ func (m *manager) deployStorageTemplate(ctx context.Context) error {
 	clusterStorageAccountName := "cluster" + m.doc.OpenShiftCluster.Properties.StorageSuffix
 	azureRegion := strings.ToLower(m.doc.OpenShiftCluster.Location) // Used in k8s object names, so must pass DNS-1123 validation
 
+	accountName, err := m.imageRegistryStorageAccountName()
```

Comment:
can we rename this to be consistent with `clusterStorageAccountName`? See above.

```suggestion
	imageRegistryStorageAccountName, err := m.imageRegistryStorageAccountName()

Reply:
Done!